lib/process-services-cloud/src/lib/task/task-form/components/task-form-cloud/task-form-cloud.component.ts

```diff
@@ -87,6 +88,14 @@ export class TaskFormCloudComponent {
     @Input()
     taskDetails: TaskDetailsCloudModel;
 
+    /** Toggle rendering of the `Open next task` checkbox. */
+    @Input()
```

Comment:
There are some MD in the doc associated t Input and Output if you modify it please regenerate them